pkgs/tools/misc/coreutils/default.nix

```
@@ -77,8 +77,6 @@ stdenv.mkDerivation rec {
       echo "int main() { return 77; }" > gnulib-tests/test-getlogin.c
     ''
   ])) + (optionalString stdenv.isAarch64 ''
-    sed '2i print "Skipping tail assert test"; exit 77' -i ./tests/tail-2/assert.sh
-
     # Sometimes fails: https://github.com/NixOS/nixpkgs/pull/143097#issuecomment-954462584
     sed '2i echo Skipping cut huge range test && exit 77' -i ./tests/misc/cut-huge-range.sh
```

file was moved https://github.com/coreutils/coreutils/blob/15925d0e5b80467c354fe27cb1205d6b6e3da11d/tests%2Fcut%2Fcut-huge-range.sh so build fails

```suggestion
    sed '2i echo Skipping cut huge range test && exit 77' -i ./tests/cut/cut-huge-range.sh
```